docs/sbctl.8.txt

```diff
@@ -113,12 +113,28 @@ EFI binary commands
                 Sign all the generated bundles.
 
 **remove-bundle** <NAME>::
-        Removes a bundle from the list.
+        Removes a bundle from the list. This does not delete the bundle itself.
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Bundles are EFI binaries which pack all the resources (initramfs, kernel, etc)
+into a single file which can be easily signed. They also don't read parameters
+from external files, which makes them more tamper-proof.
+
+When a bundle is generated, its metadata is stored into the bundle database
+(See **FILES**). Subsequent executions of *sbctl generate-bundles* will rebuild
+these bundles, so you don't need to re-specify all parameters after each system
+update.
+
+Hint: systemd-boot will automatically show entries for any bundles found in
+*$EFI/Linux/+++*+++.efi*.
```

Comment:
The proper way to write the path is `esp/EFI/Linux/+++*+++.efi` as we do on the wiki.